### ISSUE TITLE
Native window controls (close, minimize, maximize) not showing on macOS

### ISSUE THREAD

**Issue by brandonweiss** [railsware/upterm#1305]
<img width="223" alt="screen shot 2018-06-10 at 8 16 24 am" src="https://user-images.githubusercontent.com/4727/41203016-9519d2c4-6c86-11e8-9880-f0267830726e.png">

Is this a bug or is this intentional? I see some commits related custom/title bar but I can’t find any commit messages explaining the changes so I don’t know if this was supposed to happen or not and why?

**Comment by PatrickDuncan** [comment]
Hey @brandonweiss I had the same issue. Try **View > Toggle Full Screen**.

I do agree the **+** should open it in fullscreen.

**Comment by le0pard** [comment]
Should be fixed by https://github.com/railsware/upterm/pull/1306

**Comment by itssimon** [comment]
Unfortunately didn't fix it for me

**Comment by le0pard** [comment]
@itssimon did you check master branch? This fix was not published with some version.

**Comment by itssimon** [comment]
Yes, I tried on the master branch. The fullscreen mode works though.

**Comment by le0pard** [comment]
I was expected, that this PR fix this problem https://github.com/railsware/upterm/pull/1306

**Comment by brandonweiss** [comment]
I think we might be talking about different things? It seems like the commit you’re talking about changes the _behavior_ of the controls, but what I’m talking about is how the controls look.

On macOS they should be red, yellow, and green dots. But on the latest version they’re shown as a white “X“, “-”, and “+”.

**Comment by brandonweiss** [comment]
The problem is it breaks the conventions and patterns used in macOS applications. macOS applications all use the same native window controls, and in the case of non-native applications, like an Electron app, they replicate the native controls. They do this both because it’s aesthetically pleasing but also because users follow the conventions and patterns in order to understand how the application they’re using works.

Or put another way, what’s the value of changing the controls to look different from every other macOS application?

**Comment by le0pard** [comment]
Ok, now I understand a problem. Thanks. Will check, how will look native toolbar not only on Mac OS.

**Comment by le0pard** [comment]
Reverted by https://github.com/railsware/upterm/commit/dbcb73cdfa014c84f3ad2718faed6bff423ab95e

**Comment by brandonweiss** [comment]
@le0pard Thank you! 🙌 ❤️

**Comment by unbreak** [comment]
in 0.4.4 problem still exists.
